kotlin.html.markdown

```diff
@@ -169,7 +170,7 @@ fun helloWorld(val name : String) {
     println(fooData) // => DataClassExample(x=1, y=2, z=4)
 
     // Data classes have a "copy" function.
-    val fooCopy = fooData.copy(y = 100)
+    val fooCopy = fooData.copy(y=100)
```

Comment:
Thanks a lot. Python uses `parameter=default_value` style. I did not noticed kotlinlang.org uses a different style.